modules/ROOT/pages/bolt/structure-semantics.adoc

```diff
@@ -458,10 +464,35 @@ Point3D::Structure(
 
 Where the `srid` is a _Spatial Reference System Identifier_.
 
+[[structure-vector]]
+=== `Vector`
+label:new[Introduced in bolt 6.0]
```

Comment:
When referring to a section (or any block element), you can use `[role=label--new-6.0]` on the line preceding the heading, as in 
```
[role=label--new-6.0]
=== `Vector`
```
This will place the label on the same line as the heading, but won't allow you to specify the `Bolt` bit (it will only show up as `Introduced in 6.0`).